contracts/clients/src/BlsWalletWrapper.ts

```diff
@@ -221,6 +227,41 @@ export default class BlsWalletWrapper {
     return await walletContract.nonce();
   }
 
+  /** Sign an operation with an estimate of the gas required. */
+  async signWithGasEstimate(
+    operation: Omit<Operation, "gas">,
+    overhead = 0,
```

Comment:
It's not clear what this is for just based on the function sig. A docstring similar to other public functions in this class would be helpful.